src/StructurizrPHP/Core/View/RelationshipView.php

```diff
@@ -70,7 +70,7 @@ public function setPosition(?int $position) : void
         }
     }
 
-    public function getRelationship() : ?Relationship
+    public function getRelationship() : Relationship
```

Comment:
It was quite puzzling why it may be null there, but after analysis it seems to me that there is no such possibility. Correct me if I am wrong.

Reply:
I don't remember exactly, it could be a mistake, I also don't see a scenario where Relationship can be null in RelationshipView